lib/styles/editor/_menu.scss

```
@@ -16,17 +16,19 @@
   }
 
   button.neeto-editor-fixed-menu__item {
+    min-width: fit-content;
     .neeto-ui-btn__icon {
       height: 18px !important;
       width: 18px !important;
     }
   }
 
+  
   &__wrapper {
     display: flex;
     justify-content: flex-start;
     align-items: center;
-    overflow: auto;
+    overflow: scroll;
```

```suggestion
    overflow: auto;
```

This change is not necessary.
When `overflow` is scroll: The scrollbar is always shown (even if the content does not overflow).
When `overflow` is auto: The scrollbar is shown only when the content overflows the container. Hence, better UX.

Reference: https://developer.mozilla.org/en-US/docs/Web/CSS/overflow